data/json/recipes/recipe_others.json

```diff
@@ -5707,15 +5708,15 @@
   "reversible": false,
   "autolearn": true,
   "qualities":[
-    {"id":"CHEM","level":2,"amount":1},
+    {"id":"CHEM","level":1,"amount":1},
```

Comment:
Apparently it's easier than one would think. [Casein plastic can be made with a bowl and a microwave.](http://www.sciencebuddies.org/science-fair-projects/project_ideas/Chem_p101.shtml)